src/precice/Version.h.in

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+// clang-format off
+
+#define PRECICE_VERSION_MAJOR @preCICE_VERSION_MAJOR@
+#define PRECICE_VERSION_MINOR @preCICE_VERSION_MINOR@
+#define PRECICE_VERSION_PATCH @preCICE_VERSION_PATCH@
+
+#define PRECICE_VERSION "@preCICE_VERSION@"
+
+// clang-format on
+
+#define PRECICE_VERSION_GTE(major, minor, patch) (                       \
```

Comment:
Forwarding the review comment of @uekerman https://github.com/precice/precice/pull/1386#discussion_r932446877